src/tractobots_robot_localization/params/navsat_transform.yaml

```diff
@@ -11,11 +11,11 @@ delay: 0.0
 
 # If your IMU does not account for magnetic declination, enter the value for your location here. If you don't know it,
 # see http://www.ngdc.noaa.gov/geomag-web/ (make sure to convert the value to radians). This parameter is mandatory.
-magnetic_declination_radians: 0
+magnetic_declination_radians: 0  # This should be replaced with the magnetic declination for your location
 
 # Your IMU's yaw, once the magnetic_declination_radians value is added to it, should report 0 when facing east. If it
 # doesn't, enter the offset here. Defaults to 0.
-yaw_offset: 0.0
+yaw_offset: 1.5707963268
```

Comment:
navsat_transform_node assumes that +X is east, so that's where it expects to see a 0 heading. Your IMU reads 0 at magnetic north, so we need to (a) correct for magnetic declination (above), and (b) correct for the difference between east and north.